packages/common/src/components/icons/Chat.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import IconBase from 'react-icon-base';
```

Comment:
Can you explain why you installed this package we already have this:

```
import IconBase from 'react-icons/IconBase';
```

Reply:
Oh! Good point, I thought we weren't using `react-icons` in `common`. I'll change it!